core/arch/arm/plat-zynqmp/drivers/puf.c

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright 2021 Foundries.io Ltd.
+ *
+ * Jorge Ramirez-Ortiz <jorge@foundries.io>
+ */
+
+#include <initcall.h>
+#include <io.h>
+#include <kernel/delay.h>
+#include <mm/core_memprot.h>
+
+#include "puf.h"
+
+/* CSU */
+#define ISR				0x20
```

Comment:
These would fit more nicely in headers somewhere where there would be CSU section for its registers.

Reply:
for the same reason that above, I didnt feel the need to share it in some interface deficion since nobody should be reading that register out of the scope of this file...